app/src/sharedTest/java/org/oppia/app/topic/train/TopicTrainFragmentTest.kt

```diff
@@ -26,9 +31,13 @@ import javax.inject.Singleton
 class TopicTrainFragmentTest {
 
   @Test
-  fun testTopicTrainFragment_loadFragment_textIsDisplayed() {
-    ActivityScenario.launch(TopicActivity::class.java).use {
-      onView(withId(R.id.dummy_text_view)).check(matches(withText("This is dummy TextView for testing")))
+  fun testTopicTrainFragment_loadFragment_selectSkills_submitButtonIsActivated() {
```

Comment:
Other cases worth testing:
- Verifying that when no skills are selected, the button is not clickable
- Verifying that the button's clickable state becomes disabled after selecting & deselecting skills
- Verifying that clicking the submit button properly forwards the list of skills to an outgoing activity
- Verifying that the list of selected skills stay selected upon a configuration change
- Verifying that the submit button activation status stays correct for both enabled/disabled cases upon a configuration change

Reply:
Added all these working test-cases